testing/dbtest/template.go

```diff
@@ -44,6 +58,12 @@ var supportedDialects = map[string]struct{}{
 var supportedTemplates = map[string]struct{}{
 	BoundaryTemplate: {},
 	Template1:        {},
+	Boundary1000Sessions10ConnsPerSession10Template:  {},
+	Boundary1000Sessions10ConnsPerSession25Template:  {},
+	Boundary1000Sessions10ConnsPerSession50Template:  {},
+	Boundary1000Sessions10ConnsPerSession75Template:  {},
+	Boundary1000Sessions10ConnsPerSession100Template: {},
+	Boundary1000Sessions10ConnsPerSession500Template: {},
```

Comment:
this is getting cumbersome as we add additional datasets. I think we could re-work this to be a regex? Could be done in the future if this list grows though.

Reply:
I think we can wait and see how this grows, for now I'd prefer not to add a regex here.